src/modules/BlobAnalysis/BlobAnalysis.js

```diff
@@ -25,26 +25,26 @@ module.exports = function(pixels, options, priorStep){
   let unknown = new cv.Mat();
   let markers = new cv.Mat();
  
-  // gray and threshold image
+  // Gray and Threshold the image image.
```

Comment:
Why Threshold with `T` instead of `t`?
Any reason for 2 `image`?

Reply:
No reason for 2 images but I think Thresholding is a proper name. I'll fix the doubl img.